src/main/java/entities/Comment.java

```diff
@@ -1,55 +0,0 @@
-package entities;
-
```

Comment:
Why did you delete the comment entity?

Reply:
I don't recall doing that... it would be great if we could discuss how to resolve this issue in tutorial (whether I should add it back into my branch or let it merge as is)